internal/services/aws_reservation_service_test.go

```diff
@@ -0,0 +1,57 @@
+package services
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	ibClientStub "github.com/RHEnVision/provisioning-backend/internal/clients/image_builder/stubs"
+	sourcesClientStub "github.com/RHEnVision/provisioning-backend/internal/clients/sources/stubs"
+	"github.com/RHEnVision/provisioning-backend/internal/dao/stubs"
+	"github.com/RHEnVision/provisioning-backend/internal/models"
+	"github.com/RHEnVision/provisioning-backend/internal/testing/identity"
+	"github.com/stretchr/testify/assert"
+)
```

Comment:
This is missing the import, the "go test" command picks tests in random order (or unknown order to me) and this can possibly fail it it gets executed as the first test:

    _ "github.com/RHEnVision/provisioning-backend/internal/testing/initialization"

Reply:
Added :)